docs/sources/permissions/dashboard_folder_permissions.md

```diff
@@ -71,3 +71,22 @@ Result: You cannot override to a lower permission. `user1` has Admin permission
 For example if "Everyone with Editor Role Can Edit" exists in the ACL list then **John Doe** will still have Edit permission even after you have specifically added a permission for this user with the permission set to **View**. You need to remove or lower the permission level of the more general rule.
 - You cannot override permissions for users with **Org Admin Role**
 - A more specific permission with lower permission level will not have any effect if a more general rule exists with higher permission level. For example if "Everyone with Editor Role Can Edit" exists in the ACL list then **John Doe** will still have Edit permission even after you have specifically added a permission for this user with the permission set to **View**. You need to remove or lower the permission level of the more general rule.
+
+## Grant folder permissions
+
+1. In the sidebar, hover your mouse over the **Dashboards** (squares) icon and then click **Manage**.
+1. Hover your mouse cursor over a folder and click the gear icon to the right.
+1. Go to the **Permissions** tab, and then click **Add Permission**.
+1. In the **Add Permission For** dialog, select **User** or **Team**.
+1. In the second box, select the user or team to add permission for.
+1. In the third box, select the permission you want to add.
```

Comment:
I recommend listing the permissions with short definitions or linking to the topic defining definitions. This helps us be consistent, because we specify the choices in the third step, can't for the fourth (because it will be different for each user), but we can and should list the available choices for the third box. 

Apply to Grant dashboard permissions as well.